options/options_base.hpp

```diff
@@ -175,5 +175,12 @@ namespace bzn
          * @return string containing the signature
          */
         virtual std::string get_signed_key() const = 0;
+
+
+        /**
+         * Retrieve the path to the Bluzelle public key pem file
+         * @return string containing the path to the Bluzelle publi key pem file
+         */
+        virtual std::string get_bluzelle_public_key() const = 0;
```

Comment:
change to owner_public_key

Reply:
One way is enough. lets only use the conf

Reply:
Yeah, it only uses the conf.